src/web/src/browser/views/components/HomePage.tsx

```diff
@@ -1,145 +1,155 @@
+              src={getAssetUrl('partners-cloud.svg')}
+              alt=""
+            />
+            <h3>Cloud Service Providers</h3>
+            <p className="margin-bottom-4">
+              Submit your FedRAMP-compliant OSCAL System Security Plan with
+              confidence.
+            </p>
+          </div>
+        </div>
+        <div className="desktop:grid-col-4">
+          <img
+            className="float-left margin-2"
+            src={getAssetUrl('partners-assessors.svg')}
+            alt=""
+          />
+          <h3>FedRAMP Reviewers</h3>
+          <p className="margin-bottom-4">
+            Evaluate CSP submissions with an efficient workflow.{' '}
+            <a href={getUrl(Routes.developers)}>Read more...</a>
+          </p>
+        </div>
+        <div className="desktop:grid-col-4">
+          <img
+            className="float-left margin-2"
+            src={getAssetUrl('partners-agencies.svg')}
+            alt=""
+          />
+          <h3>Federal Agencies</h3>
+          <p className="margin-bottom-4">
+            Evaluate FedRAMP-approved cloud service providers with the aid of
+            structured OSCAL documentation.
+          </p>
+        </div>
       </div>
     </div>
   );
 };
+
+const HomeContent = () => {
+  return (
+    <div className="usa-prose padding-top-3">
+      <h1>Accelerate approvals</h1>
+      <p>
+        Welcome to ASAP, the upcoming FedRAMP audit validation tool. Funded by{' '}
+        <a href="https://10x.gsa.gov/">10x</a>, ASAP is comprised of the
+        following components:
+      </p>
+      <ul>
+        <li>
+          <a href="https://pages.nist.gov/OSCAL/">OSCAL</a> validation rules
+          written in Schematron format
+        </li>
+        <li>
+          This user interface, which will apply validations to a FedRAMP OSCAL
+          System Security Plan and display validation errors in-browser.{' '}
+          <a href={getUrl(Routes.validator)}>Try it out</a>.
+        </li>
+        <li>
+          Compiled Schematron rules (XSLT), which may be integrated with
+          third-party OSCAL creation/validation tools. Read our{' '}
+          <a href={getUrl(Routes.developers)}>developer documentation</a> for
+          more information.
+        </li>
+      </ul>
+      <ProcessList />
+      <h2>Why should I care?</h2>
+      <p>
+        FedRAMP audit approvals are expensive for both FedRAMP and CSPs (Cloud
+        Service Providers). The ASAP validation tool helps CSPs craft correct
+        System Security Plans, and helps the FedRAMP Audit Review Team evaluate
+        them efficiently.
+      </p>
+      <h2>What's next?</h2>
+      <ul>
+        <li>User feedback</li>
+        <li>In addition to SSP, support for POA&M, SAP, and SAR validations</li>
+      </ul>
+      <h2>Contact us</h2>
+      <p>
+        Please give us your feedback via a{' '}
+        <a href="https://github.com/18F/fedramp-automation/issues">
+          Github issue
```

Comment:
B: Minor, but since it will go upstream:

```suggestion
        Please give us your feedback via a{' '}
        <a href="https://github.com/GSA/fedramp-automation/issues">
          Github issue
```